Bots/DotNet/Skills/CodeFirst/TeamsWaterfallSkillBot/wwwroot/manifests/waterfallskillbot-manifest-1.0.json

```diff
@@ -0,0 +1,107 @@
+{
+  "$schema": "https://schemas.botframework.com/schemas/skills/skill-manifest-2.0.0.json",
```

Comment:
This manifest looks old, updated using the one we have in WaterfallSkillBot as a reference.

Reply:
we did, but not anymore.